repos/system_upgrade/el7toel8/actors/satelliteupgradedatamigration/actor.py

```diff
@@ -0,0 +1,35 @@
+import glob
+import os
+import shutil
+
+from leapp.actors import Actor
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import run
+from leapp.models import InstalledRPM
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+POSTGRESQL_DATA_PATH = '/var/lib/pgsql/data/'
+POSTGRESQL_SCL_DATA_PATH = '/var/opt/rh/rh-postgresql12/lib/pgsql/data/'
+
+
+class SatelliteUpgradeDataMigration(Actor):
+    """
+    No documentation has been provided for the satellite_upgrade_data_migration actor.
+    """
+
+    name = 'satellite_upgrade_data_migration'
+    consumes = (InstalledRPM,)
+    produces = ()
+    tags = (IPUWorkflowTag, ApplicationsPhaseTag)
+
+    def process(self):
+        has_foreman = has_package(InstalledRPM, 'foreman') or has_package(InstalledRPM, 'foreman-proxy')
+        if not has_foreman:
+            return
+
+        # remove empty PostgreSQL data from the package
+        if os.path.exists(POSTGRESQL_DATA_PATH):
+            os.rmdir(POSTGRESQL_DATA_PATH)
```

Comment:
Should this removal only happen if `POSTGRESQL_SCL_DATA_PATH` exists? If we don't need to move it, there's no reason to `rmdir`.

Reply:
nah, I plan to guard that all by "if satellite_local_postgresql", just didn't yet.